backend/API/swagger_API.yaml

```diff
@@ -0,0 +1,238 @@
+      - "application/json"
+      parameters: 
+      - in: path
+        name: studentID
+        type: integer
+        required: true
+        description: "student ID number"
+      responses: 
+        "200":
+          description: "OK"
+          schema:
+            type: "array"
+            items:
+              $ref: "#/definitions/Student"
+        "401":
+          description: "Unauthorized" 
+        "403":
+          description: "Forbidden"
+        "500":
+          description: "Internal Server Error"
+    patch:
+      tags:
+      - "Student"
+      summary: "Add additional feedback"
+      description: "Add additioinal feedback for existing student"
+      operationId: "updateStudent"      
+      consumes:
+      - "application/json"
+      produces:
+      - "integer"
+      parameters: 
+      - in: path
+        name: studentID
+        type: integer
+        required: true
+        description: "student ID number"
+      - in: "body"
+        name: "body"
+        description: "Student Feedback"
+        required: true
+        schema:
+          $ref: "#/definitions/Student"        
+      responses: 
+        "200":
+          description: "OK"      
+        "401":
+          description: "Unauthorized" 
+        "403":
+          description: "Forbidden"
+        "405":
+          description: "Method Not Allowed"
+    delete:
+      tags:
+      - "Student"
+      summary: "Remove Student & Feedback"
+      description: "Delete student, and all comments for student"
+      operationId: "removeStudent"      
+      parameters:
+      - in: path
+        name: studentID
+        type: integer
+        required: true
+        description: "student ID number"
+      responses:
+        "200":
+          description: "OK"        
+        "401":
+          description: "Unauthorized" 
+        "403":
+          description: "Forbidden"
+ 
+
+  /Student/{studentID}/comment/:
+    post:
+      tags:
+      - "Comment"
+      summary: "Get comment feedback for student"
+      description: "Get comment feedback for a particular student"
+      operationId: "commentFeedback"
+      consumes:
+      - "application/json"
+      produces:
+      - "string"
+      parameters:
+      - in: path
+        name: studentID
+        type: integer
+        required: true
+        description: "student ID number"
+      - in: "body"
+        name: "body"
+        description: "Student Feedback"
+        required: false
+        schema:
+          type: string
+      responses:
+        "200":
+          description: "OK"
+          schema:
+            type: string
+        "400":
+          description: "Bad Request"     
+        "401":
+          description: "Unauthorized" 
+        "403":
+          description: "Forbidden"
+        "500":
+          description: "Internal Server Error"  
+    get:
+      tags:
+      - "Comment"
+      summary: "Invalid operation"
+      description: "Comment delete not supported"
```

Comment:
Looks like the "delete" description got copy/pasted to the get/patch descriptions.

Reply:
Good catch, I fixed the descriptions for get/patch.